### PR TITLE
feat: Add pill buttons

### LINKED ISSUE #48: Reusable Pill Button UI
Create a reusable component we'll call "Pill Button". It should:

* Have a title
* Option for title to be bold or regular
* Have a dark theme (i.e. purple with white text) and a light theme (i.e. white. with purple text and outline)
* Have an active and inactive state
* Not a fixed width (see examples)

**Design**
Check on the screenshot below
[Link to Design](https://www.figma.com/file/4cV84H1jyMo0rJEAl3HNnJ/FightPandemics-landing-page?node-id=6638%3A29348)

**VARIATIONS**:

Light theme & dark theme

Screen Shot 2020-05-13 at 9 40 09 PM

Inactive
Screen Shot 2020-05-13 at 9 44 31 PM

### PR DESCRIPTION
**Pull request by lucianoschillagi**
**What this PR does**:
Create 4 pill buttons (regular, bold, outline, inactive)

### Checklist

- [x] Compiler warnings resolved
- [x] Linter warnings resolved
- [x] User-facing strings in Localizable.strings file
- [x] Unused code, print statements, and comments removed

**Which issue(s) this PR fixes**:

Resolves #48 

**Special notes for reviewers**:
Please use **_PillButtonPrototypeVC_** as initial ViewController

**Additional comments**:

### CODE REVIEW

File: FightPandemics/Features/Shared/PillButton/PillButton.swift [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    init(buttonType: PillButtonType, title: String) {
+        super.init(frame: .zero)
+        setTitle(title, for: .normal)
+        setTitleColor(.white, for: .normal)
+        backgroundColor = .blue
+        layer.cornerRadius = 25
+    
+        switch buttonType {
+            case .regular:
+            titleLabel?.font = UIFont.systemFont(ofSize: 22.0, weight: .regular)
```
Comment (OnyekachiSamuel) [inline FightPandemics/Features/Shared/PillButton/PillButton.swift:52]: We already have custom fonts we are using in the project, from the design identify the right fonts and use them in each of the switch cases.

File: FightPandemics/Features/Shared/PillButton/PillButtonPrototypeVC.swift [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+
+import UIKit
+
+class PillButtonPrototypeVC: UIViewController {
```
Comment (OnyekachiSamuel) [inline FightPandemics/Features/Shared/PillButton/PillButtonPrototypeVC.swift:29]: I don't really think we need this class here, the custom button you created will be used in any view controller we want.

**Review by OnyekachiSamuel — CHANGES_REQUESTED** [review]

File: FightPandemics/Features/Shared/PillButton/PillButton.swift (hunk 1 above)
Comment (lucianoschillagi) [reply FightPandemics/Features/Shared/PillButton/PillButton.swift:52]: Ok, I will do this change, thanks

File: FightPandemics/Features/Shared/PillButton/PillButtonPrototypeVC.swift (hunk 2 above)
Comment (lucianoschillagi) [reply FightPandemics/Features/Shared/PillButton/PillButtonPrototypeVC.swift:29]: Yes, I built this View Controller in order to test the pill buttons, only for this purpose. So, in my next PR I am going the remove it. Thanks Samuel!

**Pull request closed without merging**